### PR TITLE
Tines Rule - Story Jobs Clearance

### PR DESCRIPTION
**Pull request by josh-panther**
### Background

https://www.tines.com/api/audit-logs#:~:text=annotations%20were%20moved-,StoryJobsClearance,-A%20story%E2%80%99s%20scheduled

### Changes

* Tines - StoryJobsClearance rule

### Testing

```
Tines.Story.Jobs.Clearance
	[PASS] Detection Trigger
		[PASS] [rule] true
		[PASS] [title] Tines [StoryJobsClearance] performed by [user@email.com] on [tines-log-source-name].
		[PASS] [dedup] Tines [StoryJobsClearance] performed by [user@email.com] on [tines-log-source-name].
		[PASS] [alertContext] {"actor": "user@email.com", "action": "StoryJobsClearance", "tenant_id": "1337", "user_email": "user@email.com", "user_id": "7331", "operation_name": "StoryJobsClearance", "request_ip": "98.224.225.84"}
	[PASS] Tines Login
		[PASS] [rule] false
```

### CODE REVIEW

File: rules/tines_rules/tines_story_jobs_clearance.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        return False
+
+    return deep_get(event, "operation_name", default="<NO_OPERATION_NAME>") == "StoryJobsClearance"
+
+
+def title(event):
+    operation = deep_get(event, "operation_name", default="<NO_OPERATION_NAME>")
+    user = deep_get(event, "user_email", default="<NO_USER_EMAIL>")
+    tines_instance = deep_get(event, "p_source_label", default="<NO_SOURCE_LABEL>")
+
```
Comment (andrea-youwakim) [inline rules/tines_rules/tines_story_jobs_clearance.py:17]: please remove this unnecessary whitespace

File: rules/tines_rules/tines_story_jobs_clearance.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Filename: tines_story_jobs_clearance.py
+RuleID: Tines.Story.Jobs.Clearance
+DisplayName: "Tines Story Jobs Clearance"
+Enabled: true
+LogTypes:
+    - Tines.Audit
+Tags:
+  - Tines
+Severity: Low
+Description: "A user has cleared story jobs"
```
Comment (andrea-youwakim) [inline rules/tines_rules/tines_story_jobs_clearance.yml:11]:
```suggestion
Description: "A Tines User has cleared story jobs."
```

**Review by arielkr256 — APPROVED** [review]
